### PR TITLE
Login

### PR DESCRIPTION
**Pull request by rov9**
Please review.Live url:https://cocky-pare-3dfb7c.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ravikumar0403) [inline index.html:7]: Change the title to be relevant with your page

File: style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+.logo{
+    height: 16rem;
+    display: block;
+    margin: auto;
+}
+
+
+#name,#email,#password,#mob-no{
```
Comment (ravikumar0403) [inline style.css:17]: Not mandatory but it's better to avoid id for styling and use class

File: style.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    display: block;  
+    margin: 1rem auto;  
+    border-radius: 5px;
+    box-shadow: 0 4px 10px rgba(0, 0, 0, 0.01);
```
Comment (ravikumar0403) [inline style.css:21]: Use variables for color

**Review by ravikumar0403 — CHANGES_REQUESTED** [review]

**Comment by rov9** [comment]
Authorization Page-Login

**Pull request merged**